### ISSUE TITLE
1.16 Release Notes: "Known Issues"

### ISSUE THREAD

**Issue by saschagrunert** [kubernetes/kubernetes#81930]
This issue is a bucket placeholder for collaborating on the "Known Issues" additions for the 1.16 Release Notes. If you know of issues or API changes that are going out in 1.16, please comment here so that we can coordinate incorporating information about these changes in the Release Notes.

/assign @cartyc @kcmartin @onyiny-ang @paulbouwer

/sig release
/milestone v1.16

**Comment by saschagrunert** [comment]
@onyiny-ang can you please set the milestone for me?

**Comment by saschagrunert** [comment]
It seems that we have to document a known issue related to the iptables problematic.

**Comment by saschagrunert** [comment]
Proposal for the *Known Issues* section:
````
- kube-proxy is currently incompatible with systems running `iptables` newer than version 1.8.0.
  Please note that this affects all versions of Kubernetes and not only v1.16.0.
  As a workaround, please update the host systems to use the `legacy` backend per default:
  ```
  update-alternatives --set iptables /usr/sbin/iptables-legacy
  update-alternatives --set ip6tables /usr/sbin/ip6tables-legacy
  update-alternatives --set arptables /usr/sbin/arptables-legacy
  update-alternatives --set ebtables /usr/sbin/ebtables-legacy
  ```
````

cc @dims

**Comment by saschagrunert** [comment]
cc @thockin :point_up:

**Comment by praseodym** [comment]
> It seems that we have to document a known issue related to the iptables problematic.

https://github.com/kubernetes/website/pull/16271 is a docs PR for this and has some more context + instructions for RHEL/CentOS/Fedora, maybe we should link there instead (once that's merged)?

**Comment by danwinship** [comment]
> - kube-proxy is currently incompatible with systems running `iptables` newer than version 1.8.0.

That's not accurate. (I guess you're basically paraphrasing the title of #71305, but that title isn't accurate.)

There are two problems:
1. #71305 - If you are using the official kube-proxy container images (including, if you are using kubeadm) on a system with iptables 1.8, then you need to configure the system to use iptables in legacy mode.
2. #82361 - If you are on a system with iptables 1.8.0-1.8.2, you have to use iptables in legacy mode even if you aren't using the official kube-proxy container images.

The tl;dr being "if you're on a system with iptables 1.8, then use legacy mode, for now".

**Comment by saschagrunert** [comment]
Thanks for the input @danwinship and @praseodym.

So then we I guess we could write up:
````
- Systems running `iptables` newer than version 1.8.0 should switch to legacy mode for now.
  Please note that this affects all versions of Kubernetes and not only v1.16.0.
  For more detailed information about the issue and how to apply a workaround, please refer to the 
  [official documentation](https://...)
````

WDYT?

**Comment by danwinship** [comment]
> Systems running `iptables` newer than version 1.8.0

"Systems running `iptables` 1.8.0 or newer". (ie, clarify that 1.8.0 is bad)

> should switch to legacy mode

"should run iptables in legacy mode" (ie, it's iptables that needs "legacy mode", not kubernetes)

**Comment by saschagrunert** [comment]
> > Systems running `iptables` newer than version 1.8.0
> 
> "Systems running `iptables` 1.8.0 or newer". (ie, clarify that 1.8.0 is bad)
> 
> > should switch to legacy mode
> 
> "should run iptables in legacy mode" (ie, it's iptables that needs "legacy mode", not kubernetes)

Like this:
> Systems running `iptables` 1.8.0 or newer should start it in legacy mode.

**Comment by logicalhan** [comment]
How's this:
```
Etcd and KMS plugin health checks are not exposed in the new livez and readyz endpoints. This will be fixed in 1.16.1.
```

**Comment by saschagrunert** [comment]
> Regarding issue (#82713):
> 
> ```
> Etcd and KMS plugin health checks are not exposed in the new livez and readyz endpoints. This will be fixed in 1.16.1.
> ```

Hey @logicalhan, I added the note to the draft, thanks.

@justincormack, @danwinship, @praseodym are you fine with that entry for the iptables topic or should we not it at all?
````
- Systems running `iptables` 1.8.0 or newer should start it in legacy mode.
  Please note that this affects all versions of Kubernetes and not only v1.16.0.
  For more detailed information about the issue and how to apply a workaround, please refer to the 
  [official documentation](https://...)
````

**Comment by saschagrunert** [comment]
The final release notes draft is published. Closing this for now. Feel free to reach us out on slack (#release-notes) if anything goes wrong.

/close
